bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -88,7 +96,7 @@
     /**
      * {@link Map} of module type UIDs to rules where these module types participated.
      */
-    private Map<String, Set<String>> mapModuleTypeToRules = new HashMap<String, Set<String>>();
+    private static Map<String, Set<String>> mapModuleTypeToRules = new HashMap<String, Set<String>>();
```

Comment:
Why did this (and mtManager, tManager and compositeFactory) become static? Is this really necessary?